e2e/live/testdata/migrate-case-2a/namespace.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
Suggest modifying and adding the "Copyright **2022** The Kubernetes Authors..." here and in other files

Reply:
Are these files copied from a Kubernetes repo? If not, why are we using the Kubernetes Authors copyright?

Reply:
The files in this directory were copied over from `e2e/live/testdata/migrate-case-1a` as is. A new directory is used since the e2e shell scripts doesn't clean up the testdata directory after being run, hence it seemed like a cleaner approach for me to have a separate testdata folder for the new migrate e2e test case. Let me know how the license header should be handled if we do stick with having the new test case in the shell script, rather than the Go test harness.

Reply:
so as Morten mentioned, shall we update this to kpt Authors?

Reply:
Given that these files were copied directly from e2e/live/testdata/migrate-case-1a, and they contained the Kubernetes authors as copyright, it seems like we should maintain the Kubernetes authors still?